pkgs/top-level/haskell-packages.nix

```diff
@@ -77,7 +77,7 @@ in {
         # aarch64 ghc865Binary gets SEGVs due to haskell#15449 or similar
         # Musl bindists do not exist for ghc 8.6.5, so we use 8.10.* for them
         else if stdenv.isAarch64 || stdenv.targetPlatform.isMusl then
```

Comment:
You can drop the `aarch64-darwin` condition above, since `stdenv.isAarch64` will also catch that.

Reply:
Yes, that makes sense.